explorer.service

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=BlockSkout Daemon
```

Comment:
Let's create a sample explorer.sample.service and add explorer.service to .gitignore, so that we can make changes

Reply:
Alright. Make sense.